protected/humhub/modules/admin/models/forms/ApproveUserForm.php

```diff
@@ -294,6 +353,26 @@ public function setDeclineDefaults()
         Yii::$app->i18n->autosetLocale();
     }
 
+    /**
+     * Returns the default send message. If not parameters set, the placeholder names are returned.
+     *
+     * @param string $userDisplayName
+     * @param string $adminDisplayName
+     * @return string
+     */
+    public static function getDefaultSendMessageMailContent($userDisplayName = '{displayName}', $adminDisplayName = '{AdminName}')
+    {
+        return Yii::t('AdminModule.user', "Hello {displayName},\n\n" .
+            "Your account creation is under review.\n" .
```

Comment:
@Semir1212 Can you please review, this mail text?

Reply:
Fine as is